about.md

```diff
@@ -6,294 +6,48 @@ title: About MDAnalysis
 <img src="{{ site.baseurl }}/public/mdanalysis-logo_square.png"
 style="float: right" alt="MDAnalysis" width="30%"/>
 
-## Mission
+MDAnalysis is an open-source Python library for analyzing molecular dynamics simulations and biomolecular structures. It enables high-performance, reproducible computational studies, supporting researchers across academia and industry.
```

Comment:
Too narrow. This is just the library. The website is about the MDAnalysis Project.

As the Mission says, the library is important, but there's more to the project.